src/vs/workbench/services/extensions/common/abstractExtensionService.ts

```diff
@@ -201,9 +201,13 @@ export abstract class AbstractExtensionService extends Disposable implements IEx
 					// We need to disconnect the management connection before killing the local extension host.
 					// Otherwise, the local extension host might terminate the underlying tunnel before the
 					// management connection has a chance to send its disconnection message.
-					await this._remoteAgentService.endConnection();
-					await this._doStopExtensionHosts();
-					this._remoteAgentService.getConnection()?.dispose();
+					try {
+						await this._remoteAgentService.endConnection();
+						await this._doStopExtensionHosts();
+						this._remoteAgentService.getConnection()?.dispose();
+					} catch {
+						this._logService.warn('Error while disconnecting remote agent');
```

Comment:
Nit: would it make sense to include the error message?

Reply:
Probably, though network errors are generally already logged internally by the connection infrastructure so we don't miss a whole lot at the moment